sdk/documenttranslation/Azure.AI.DocumentTranslation/src/Azure.AI.DocumentTranslation.csproj

```diff
@@ -0,0 +1,31 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+  
+  <PropertyGroup>
+    <Description>Translator is a cloud-based machine translation service and is part of the Azure Cognitive Services family of cognitive APIs used to build intelligent apps. Translator is easy to integrate in your applications, websites, tools, and solutions. It allows you to add multi-language user experiences in 90 languages and dialects. And it can be used on any hardware platform with any operating system for text translation.</Description>
+    <AssemblyTitle>Microsoft Azure.AI.DocumentTranslation client library</AssemblyTitle>
+    <Version>1.0-preview.1</Version>
```

Comment:
looks like this might be `1.0.0-preview.1` same for the version on the changelog